### PR TITLE
NEW Add hook printTabsHeader

### PR DESCRIPTION
**Pull request by defrance**
Add number of collected tabs 
Add trigger based on tabs name
add hook at end of tabs and at the footer of page

### CODE REVIEW

**Comment by defrance** [comment]
seems travis not up to date

File: htdocs/core/lib/functions.lib.php [hunk 1]
```diff
@@ -702,8 +702,9 @@ function dol_fiche_head($links=array(), $active='0', $title='', $notab=0, $picto
  */
 function dol_get_fiche_head($links=array(), $active='0', $title='', $notab=0, $picto='', $pictoisfullpath=0)
 {
-	global $conf,$langs;
-
+	global $conf,$langs, $hookmanager;;
```
Comment (simnandez) [inline htdocs/core/lib/functions.lib.php]: $hookmanager;; = many ;

**Comment by eldy** [comment]
There is a property hookmanager->contextarray that can be set just before calling the hook.
You can use it to provide data specific to the tab.

**Comment by defrance** [comment]
$hookmanager->initHooks(xxx) fill the contextarray property
otherwise, i can send it as parameters of  $hookmanager->executehooks
propose another change

**Comment by eldy** [comment]
You're right. The ->contextarray and $parameters['context'] is reserved for hook framework.

So you must use 
$parameters=array( 'tabname' => $mytabname);

to provide more information for external hook functions.

**Comment by defrance** [comment]
sorry so "change made like eldy does" ;-)

**Pull request merged**